clouds/snowflake/.~lock.results.csv#

```diff
@@ -0,0 +1 @@
+,dean,thinkpad,27.03.2024 10:56,file:///home/dean/.config/libreoffice/4;
```

Comment:
We can clean this file and include a rule in .gitignore (not necessarily now to avoid delaying the release)

Reply:
I'll be adding it into a separate PR